src/__tests__/clients/db/LabRepository.tests.ts

```diff
@@ -0,0 +1,151 @@
+/* eslint "@typescript-eslint/camelcase": "off" */
```

Comment:
Thank you so much for adding tests for the entire file! 💛

Reply:
Welcome!, thanks for the code review!